experiments/integrated/ozark/conservation/ozark_conservation.jl

```diff
@@ -200,6 +249,9 @@ Y.soil.ρe_int =
         T_0,
         Ref(land.soil.parameters),
     )
+
+Y.soilco2.C .= FT(100)
```

Comment:
set equal to C, define above?

Reply:
Thanks!
I've set it to atmospheric co2, same as ozark.jl and tutorial